x-pack/plugins/security_solution/public/resolver/test_utilities/simulator/index.tsx

```diff
@@ -146,6 +147,30 @@ export class Simulator {
       .forEach((wrapper) => wrapper.setState({ width: 10000, height: 10000 }));
   }
 
+  public requestNewTree(documentIdToFetch: string) {
```

Comment:
I was thinking that these tests would all interact with the UI just the user does. What is your reasoning for having these methods?

Reply:
Yea, didn't like this pattern at all. Updated